shared/naturalcrit/splitPane/splitPane.jsx

```diff
@@ -89,6 +106,21 @@ const SplitPane = createClass({
 			userSetDividerPos : newSize
 		});
 	},
+
+	liveScrollToggle : function(e) {
+		const flipLiveScroll = !this.state.liveScroll;
+		const toggle = e.target;
+		const toggleDiv = toggle.parentElement;
+		if(flipLiveScroll) {
+			toggle.className = 'fas fa-lock';
+			toggleDiv.className = 'arrow lock';
+		} else {
+			toggle.className = 'fas fa-unlock';
+			toggleDiv.className = 'arrow unlock';
+		}
+		window.localStorage.setItem('liveScroll', String(flipLiveScroll));
+		this.setState({ liveScroll: flipLiveScroll });
```

Comment:
Same thing about not modifying the DOM here. Sometimes it's ok to _read_ the contents of the DOM. But even then, there is a React way to do it.

So similar thing. Here, the logic would be:

Click toggle button -> toggle the state -> Render the updated component based on the state value.